### ISSUE TITLE
The model does not detect large faces

### ISSUE THREAD

**Issue by dana170** [deepcam-cn/yolov5-face#37]
I am training the model on the dataset with mixed images (small faces, and large faces). When I use the best weights and detect faces on the test set, there are no detections on large faces. I have tried using all versions including those with P6 output block.

**Comment by dana170** [comment]
I am afraid I cannot share the dataset due to the ethical concerns. But I can send you the blurred version of two images (containing small faces and the one with large faces) to illustrate the difference in size between faces to your email address. If it is ok, where can I find your email address?

**Comment by derronqi** [comment]
we open a new training dataset, improve the large face detection.
